health/notifications/matrix/README.md

```diff
@@ -0,0 +1,42 @@
+# matrix.org - riot.im
+
+Send notifications to matrix network rooms.
+
+The requirements for this connector are
+1. The url of the homeserver (https://homeserver:port)
```

Comment:
@joelhans markdown linter doesnt like this URL

Reply:
I know, but it's not real URL :) it's example

Reply:
It should be in an inline code block to prevent the link-checker from trying to follow it.

```suggestion
1. The url of the homeserver (`https://homeserver:port`)
```